A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/CDU/A320_Neo_CDU_InitPage.js

```diff
@@ -51,13 +50,25 @@ class CDUInitPage {
                 }
             };
         }
+        let alignOption;
+        var events = [];
+        for (var property in document.documentElement) {
+        var match = property.match(/^on(.*)/)
+        if (match) {
+            events.push(match[1]);
+            }
+        }
+        // console.log(events.join('\n'))
+        if (mcdu.flightPlanManager.getOrigin()) {
+            alignOption = "IRS INIT>"
+            console.log(mcdu.flightPlanManager.getOrigin().ident);
+        }
```

Comment:
This is badly indented and hard to understand. Please fix the indentation ;) Also, never use `var`.

Please also remove `console.log`s from release code.